explorer/interpreter/interpreter.cpp

```diff
@@ -2505,6 +2509,10 @@ auto Interpreter::Step() -> ErrorOr<Success> {
 auto Interpreter::RunAllSteps(std::unique_ptr<Action> action)
     -> ErrorOr<Success> {
   if (trace_stream_->is_enabled()) {
+    *trace_stream_ << "--- running all steps for '" << action->kind_string()
+                   << "'\n'''\n";
+    action->Print(trace_stream_->stream());
+    *trace_stream_ << "\n''' ---> \n";
```

Comment:
```suggestion
    *trace_stream_ << "--- running all steps for '" << action->kind_string()
                   << "'\n```\n";
    action->Print(trace_stream_->stream());
    *trace_stream_ << "\n``` ---> \n";
```